dandischema/models.py

```diff
@@ -58,8 +64,12 @@
 )
 ASSET_UUID_PATTERN = r"^dandiasset:" + UUID_PATTERN
 VERSION_PATTERN = r"\d{6}/\d+\.\d+\.\d+"
-DANDI_DOI_PATTERN = rf"^10.(48324|80507)/dandi\.{VERSION_PATTERN}"
-DANDI_PUBID_PATTERN = rf"^DANDI:{VERSION_PATTERN}"
+DANDI_DOI_PATTERN = (
+    rf"^10\.{DATACITE_DOI_ID_PATTERN}/{ID_PATTERN.lower()}\.{VERSION_PATTERN}"
+)
+DANDI_PUBID_PATTERN = rf"^{ID_PATTERN}:{VERSION_PATTERN}"
+DANDI_NSKEY = "dandi"  # Namespace for DANDI ontology
```

Comment:
Should be "dandi" no matter what? or would ID_PATTERN.lower() work here too?

Reply:
Are you talking about `DANDI_NSKEY`? If you are the answer is yes, according to @yarikoptic, the namespace for DANDI ontology is not going to change. `ID_PATTERN.lower()` can have a different value than `"dandi"`, so it will not work.